### PR TITLE
AdminController: adding unit tests

### LINKED ISSUE #609: AdminController: adding unit tests

### PR DESCRIPTION
**Pull request by mgmccarthy**
fixes #609

### CODE REVIEW

**Comment by tonysurma** [comment]
pinging @mheggeseth @MisterJames for a quick review before we move forward. Thanks!

File: AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs [hunk 1]
```diff
@@ -153,27 +152,24 @@ public IActionResult Register()
...
         [AllowAnonymous]
-        public async Task<IActionResult> ConfirmEmail(string userId, string code)
+        public async Task<IActionResult> ConfirmEmail(string userId, string token)
         {
-            if (userId == null || code == null)
-            {
+            if (userId == null || token == null)
                 return View("Error");
-            }
+
```
Comment (BillWagner) [inline AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs]: We should add this to the style guide, but we do prefer having the braces around if and else clauses, even for single statement clauses.

File: AllReadyApp/Web-App/AllReady/Controllers/AdminController.cs [hunk 2]
```diff
@@ -88,28 +90,23 @@ public IActionResult DisplayEmail()
...
         [AllowAnonymous]
-        public async Task<IActionResult> ConfirmEmail(string userId, string code)
+        public async Task<IActionResult> ConfirmEmail(string userId, string token)
         {
-            if (userId == null || code == null)
-            {
+            if (token == null)
                 return View("Error");
-            }
+
```
Comment (BillWagner) [inline AllReadyApp/Web-App/AllReady/Controllers/AdminController.cs]: Same comment: Please leave the braces in place here.

**Comment by BillWagner** [comment]
Left a few comments. And create this issue: https://github.com/HTBox/htbox.github.io/issues/4

Once the comments are addressed, and the merge conflicts resolved, this is ready to merge.

File: AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs (hunk 1 above)
Comment (mgmccarthy) [inline AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs]: @BillWagner crap. I've been coding all along to take the curly braces out of single line statement clauses. Going forward I'll adhere to that approach

Comment (BillWagner) [inline AllReadyApp/Web-App/AllReady/Controllers/AccountController.cs]: I feel bad because we should have documented that expectation. We'll get there.

**Comment by mgmccarthy** [comment]
@BillWagner, how do I go about handling these merge conflicts? The regular steps for rebasing from master?

**Comment by BillWagner** [comment]
@mgmccarthy Yes, that's right. Rebase from master (which will show the merge conflicts) and resolve them.

**Comment by mgmccarthy** [comment]
pinging @BillWagner. All merge conflicts fixed. This is ready to go.

**Pull request merged**